app/views/admin/notes/_form.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_with model: [:admin, note], local: true do |f| %>
+  <%= f.hidden_field :submission_id %>
```

Comment:
I think this should be part of the path. The [shallow nesting](https://guides.rubyonrails.org/routing.html#shallow-nesting) is the Rails term for this kind of stuff.